torch/sparse/_triton_ops.py

```diff
@@ -89,137 +91,143 @@ def valid_grid_dim(g, mg):
         kernel(grid, *sliced_tensors)
 
 if _has_triton():
-    import triton
```

Comment:
I still think you need to remove the `_has_triton()` check above. Otherwise importing this file will import triton (which is what you're trying to avoid!

Reply:
np. The test above should make sure we don't miss on in the future!